tests/simple/error.uplc.expected

```diff
@@ -1,11 +0,0 @@
-<generatedTop>
```

Comment:
I know that uplc emits an error on this that's emitted on stderr. @ChristianoBraga and @ehildenb should we capture the output of stderr when we capture expected results?

Reply:
Yes, just have to change the semantics to generate what has caused the error. In the `error.uplc` is empty 
```
An error has occurred:  User error:
The provided Plutus code called 'error'.
``` 
but `(program 0.0.0 [[(builtin divideInteger) (con integer 1)] (con integer 0)])` produces 
```
An error has occurred:  User error:
The provided Plutus code called 'error'.
Caused by: (divideInteger 1 0)
```